proxy/proxytest/proxytest.go

```diff
@@ -42,6 +43,7 @@ func newTestProxy(fr filters.Registry, routingOptions routing.Options, proxyPara
 		routingOptions.Predicates = []routing.PredicateSpec{
 			loadbalancer.NewGroup(),
 			loadbalancer.NewMember(),
+			traffic.New(),
```

Comment:
Given your last comment, is this needed still? I don't fully understand what it's used for.